doc/sphinx/index.rst

```diff
@@ -1,13 +1,8 @@
-.. GPflow documentation master file, created by
```

Comment:
Currently, the index looks a little bit empty. Are we planning to change this? Can we point it to `intro.html` in the meantime? Alternatively, can we make intro the index?